app/javascript/application.js

```diff
@@ -1,3 +1,19 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
-import "./controllers"
+// import "@hotwired/turbo-rails"
+// import "./controllers"
+
+import React from 'react';
```

Comment:
- [ ] DEar @Peter1907, though your project is great, Please consider removing commented codes as it will make your codebase messy. Remember, if you ever want to see the code you have written before, you can use your commit history.

Reply:
Okay sure thank you. But I think this could've been an optional suggestion? :shrug: :joy:

Reply:
I wish I could @Peter1907, But it's among the primary requirements in all of the projects, and projects with commented codes are not allowed to be approved.